### PR TITLE
Salt Mine edits

### PR DESCRIPTION
**Pull request by Loquacity**
Updates per feedback on https://github.com/SUSE/doc-susemanager/pull/474

### CODE REVIEW

**Comment by Loquacity** [comment]
Added that last bit. Happy for me to merge now?

**Comment by moio** [comment]
Reminder: this has to be backported to 3.2 as well :wink:

File: modules/salt/pages/salt-rate-limiting.adoc [hunk 1]
```diff
@@ -71,9 +71,11 @@ Defaults to 1 second.
 == Disabling the Salt Mine
 
 In older versions, {productname} used a tool called Salt mine to check minion availability.
-The Salt mine would cause minions to ping the server every hour, which created significant load.
+The Salt mine would cause minions to contact the server every hour, which created significant load.
 With the introduction of a more efficient mechanism in {productname}{nbsp}3.2, the Salt mine is no longer required.
-Instead, the {productname} server uses Taskomatic to ping only the minions that appear to have been offline for twelve hours or more, with all minions being pinged at least once in every twenty four hour period by default.
+Instead, the {productname} server uses Taskomatic to ping only the minions that appear to have been offline for twelve hours or more, with all minions being contacted at least once in every twenty four hour period by default.
+You can adjust this by changing the [systemitem]``web.system_checkin_threshold`` parameter in [path]``rhn.conf``.
+The value is expressed in days, and the default value is [systemitem]``1``.
```
Comment (keichwa) [inline modules/salt/pages/salt-rate-limiting.adoc]:
```suggestion
The value is expressed in days, and the default value is [literal]``1``.
```
Markup is a hopeless case with adoc.  Maybe, we'd better start using just ``value`` or ```value``` for value (this is: single or double backtics--I do not know whether there is a difference or not).  I've no clue how you can escape backtics with markdown...

**Comment by Loquacity** [comment]
> Reminder: this has to be backported to 3.2 as well wink

Please review: https://github.com/SUSE/doc-susemanager/pull/499

**Pull request merged**